code/model/Feedback.php

```diff
@@ -1,21 +1,29 @@
 <?php
 class Feedback extends DataObject
 {
-	static $db = array(
+	/**
+	 * @var array
+	 */
+	public static $db = array(
 		'Rating' => 'Boolean',
 		'Comment' => 'Text',
-		'URL' => 'Text'
+		'URL' => 'Text',
+		'IP' => 'Text'
 	);
 
-	static $has_one = array(
+	/**
+	 * @var array
+	 */
+	public static $has_one = array(
 		'Page' => 'SiteTree'
 	);
 
-
-	static $summary_fields = array(
+	/**
+	 * @var array
+	 */
+	public static $summary_fields = array(
 		'Rating',
 		'Comment',
 		'URL'
 	);
```

Comment:
We should add canX methods here with some default permissions which is good practice otherwise only admins can access these dataobjects.